### PR TITLE
Add the Arabic.ahk file

### PR DESCRIPTION
**Pull request by anthonyyoussef01**
- Basically just the opposite of the Franco.ahk

### CODE REVIEW

File: Arabic.ahk [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+lastPressed := ""
+return
+
+$+5::
+if (lastPressed = "-5"){
+    Send, {BackSpace}5
+    lastPressed := "+5"
+}
+else if (lastPressed = "+5"){
+    Send, {BackSpace}%
```
Comment (popcar2) [inline Arabic.ahk]: This should be `% and not just %

**Review by popcar2 — CHANGES_REQUESTED** [review]
I'm not 100% sure why you'd want to write arabic --> franco but it looks fine. Just need to fix this one typo that causes an error.

File: Arabic.ahk (hunk 1 above)
Comment (anthonyyoussef01) [reply Arabic.ahk]: 😂

**Comment by anthonyyoussef01** [comment]
I'd love it if you could take a look at my Qamous project (https://github.com/anthonyyoussef01/Qamous)

**Pull request merged**